src/solvers/flattening/boolbv_overflow.cpp

```diff
@@ -112,7 +112,8 @@ literalt boolbvt::convert_overflow(const exprt &expr)
 
     bvt bv_ext=bv_utils.extension(bv0, new_size, rep);
 
-    bvt result=bv_utils.shift(bv_ext, bv_utilst::shiftt::LEFT, bv1);
+    bvt result=bv_utils.shift(
+      bv_ext, bv_utilst::shiftt::SHIFT_LEFT, bv1);
```

Comment:
If it fits on one line, that would be better to my eyes.

Reply:
done